package.json

```diff
@@ -62,7 +62,7 @@
     "better-than-before": "1.0.0",
     "chai": "3.4.1",
     "chokidar-cli": "1.2.0",
-    "conventional-changelog-core": "1.0.1",
+    "conventional-changelog-core": "1.8.0",
```

Comment:
I couldn't test the scenario I was running into without upgrading.

Also it will only effect your tests since you had it listed as a devDependency.